src/components/table/Table.tsx

```diff
@@ -35,12 +39,12 @@ function Table<T extends { id: number }>({
           </tr>
         </thead>
         <tbody>
-          {data.map((row) => (
+          {data.map((row, rowIndex) => (
             <tr key={row.id}>
               {columns.map((column) => (
                 <td key={`${row.id}-${String(column.accessor)}`}>
                   {column.render
```

Comment:
kindly replace ~`column.render`~ to `column?.render`. This will ensure your code don't break if `render` don't exist.